scripts/validate.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+export $(grep -v -e '^#'  -e '^MNEMONIC' .env | xargs -0)
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+echo "Validating Timestamp between Native and $NETWORK"
+
+npx hardhat run scripts/validateTimestamp.js  --network $NETWORK
```

Comment:
run the forge command in here as well

Reply:
needs some extra flags and sometimes it does not depending on the network. Better to run the command from command line itself

Reply:
including running simulations etc and index of the ledger device could change, not hardcoding it for now @SamAg19

Reply:
@yohanelly95 There is no point in using the shell script then. We need to incorporate validateTimestamp logic inside the deployer script